dash_app.py

```diff
@@ -178,17 +177,20 @@ def make_cell(item, skin_tone, gender, font_size):
         target = item
     if additional_emojis:
         return [
-            wrap_emoji(target, font_size),
-            dbc.Button('More',
-                       id={
-                           'type': 'more-button',
-                           'index': item['text']
-                       },
-                       className="me-1",
-                       size='sm',
-                       outline=True,
-                       color='dark',
-                       style={'margin-top': '1em'}),
+            html.Div(
+                children=[
+                    html.Div(wrap_emoji(target, font_size)),
+                    dbc.Button('More',
+                               id={
+                                   'type': 'more-button',
+                                   'index': item['text']
+                               },
+                               className="btn-secondary btn-sm")
```

Comment:
This regresses the class, size, outline ,and color keywords back to the `className` that doesn't really work.